backend/services/interfaces/IImageUploadService.ts

```diff
@@ -0,0 +1,17 @@
+import { FileUpload } from "graphql-upload";
+
+export interface UploadedImage {
+  image: string;
+  path: string;
+}
+
+interface IImageUploadService {
+  /**
+   * Upload an image to Firebase.
+   * @param file the file to upload
+   * @returns a link and filename for the uploaded image
```

Comment:
nit: based on the UploadedImage interface, it might be more accurate to say
```suggestion
   * @returns a link and file path for the uploaded image
```